distribution/docs/src/main/resources/content/_metadataAttributes/core-attributes-table.adoc

```diff
@@ -55,7 +55,7 @@
 |[[_location]]location
 |The primary geospatial location of the resource.
 |Geometry
-|Valid Well Known Text (WKT) per http://www.opengeospatial.org/standards/wkt-crs +
+a|Valid Well Known Text (WKT) per http://www.opengeospatial.org/standards/wkt-crs {external-link} +
```

Comment:
I am thinking this `a` means something now....

Reply:
it's tells asciidoctor that there is asciidoctor formatted content in this cell